xml/hardening.xml

```diff
@@ -3445,6 +3445,327 @@ Account Expires: Never</screen>
+     when you restrict &rootuser; logins to
+     <literal>tty1</literal>, then a &rootuser; login in the
+     graphical session will be denied.
+    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-sec-prot-restrict-root-ssh">
+    <title>Restricting SSH Logins</title>
+    <para>
+     By default the &rootuser; user is also allowed to login
+     into a machine remotely via the SSH network protocol (if the SSH port is
+     not blocked by the firewall). To restrict this possibility the following
+     change of the OpenSSH configuration needs to be performed:
+    </para>
+    <procedure>
+     <step>
+      <para>Edit <filename>/etc/ssh/sshd_config</filename> and adjust the following parameter:</para>
+      <screen>PermitRootLogin no</screen>
+     </step>
+     <step>
+      <para>Restart the <systemitem class="service">sshd</systemitem> service to make the changes effective:</para>
+      <screen>systemctl restart sshd.service</screen>
+     </step>
+    </procedure>
+    <note>
+     <para>
+      Using the PAM module <literal>pam_securetty</literal> is not suitable in
+      case of OpenSSH, because SSH logins not necessarily go through the PAM
+      stack during the auth phase (e.g. when using SSH public key authentication).
+      Also an attacker could be able to differentiate between a wrong password
+      and a successful login that was only rejected later on by policy.
+     </para>
+    </note>
+   </sect2>
+
+  </sect1>
+  <sect1 xml:id="sec-sec-prot-inactivity-logout">
+   <title>Setting an Inactivity Timeout for Interactive Shell Sessions</title>
+   <para>
+    It can be a good idea to terminate an interactive shell session after a
+    certain time of inactivity, for example to:
+   </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+      prevent open, unguarded sessions.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       avoid waste of system resources.
+      </para>
+     </listitem>
+    </itemizedlist>
+   <para>
+    By default, there is no inactivity timeout for shells. Nothing will
+    happen if a shell stays open unused for days or even years. It is possible
+    to configure most shells in a way that idle sessions terminate
+    automatically after a certain amount of time. The following example
+    shows how to set an inactivity timeout for a number of
+    common types of shells.
+   </para>
+   <para>
+    The inactivity timeout can be configured for login shells only or for all
+    interactive shells. In the latter case the inactivity timeout is running
+    individually for each shell instance. This means that timeouts will
+    accumulate. When a sub or child shell is started then the timeout runs for
+    the sub or child shell and only afterwards will the
+    timeout of the parent continue running.
+   </para>
+   <para>
+    The following table contains configuration details for a selection of
+    common shells shipped with SLE:
```

Comment:
I suspect that there is an entity for "SLE" here.

Reply:
I used the `&productname;` in commit 18deb0107